src/neural/cuda/network_cudnn.cc

```diff
@@ -207,22 +207,22 @@ class CudnnNetwork : public Network {
     // Hardcoded right now:
     //  3 for input and residual block convolutions.
     //  1 for policy and value head convolutions.
-    processConvBlock(weights.input, true, 3);
+    weights.input.FoldBN(3);
     for (int i = 0; i < numBlocks_; i++) {
       if (weights.residual[i].has_se) {
         has_se_ = true;
       }
-      processConvBlock(weights.residual[i].conv1, true, 3);
-      processConvBlock(weights.residual[i].conv2, true, 3);
+      weights.residual[i].conv1.FoldBN(3);
+      weights.residual[i].conv2.FoldBN(3);
     }
     if (conv_policy_) {
-      processConvBlock(weights.policy1, true, 1);
+      weights.policy1.FoldBN(3);
```

Comment:
Was the old code wrong here?  It still had 1 rather than 3...

Reply:
Yes, the old code looks wrong as we create the convolution as 3x3 when conv_policy_ is enabled: https://github.com/LeelaChessZero/lc0/blob/master/src/neural/cuda/network_cudnn.cc#L325

Reply:
As discussed on discord, while it was 'wrong' there is no change in behavior between 1 and 3 in practice so long as the actual filter size is 3.